### PR TITLE
[INLONG-902][Doc] Add MySQL to ClickHouse example document for data sync

### LINKED ISSUE #902: [Improve][Doc] Add MySQL to ClickHouse example document for data sync
### Description

Add `MySQL` to the `ClickHouse` example document for data sync.

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by haifxu**
### Prepare a Pull Request

- Fixes #902 

### Motivation

Add MySQL to the ClickHouse example document for data sync

### Modifications

Add MySQL to the ClickHouse example document for data sync

### Verifying this change
English:
mysql_clickhouse_example

Chinese:
mysql_clickhouse_example_zh

### CODE REVIEW

File: docs/quick_start/data_sync/mysql_clickhouse_example.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+The ClusterTags selects the newly created `default_cluster`, the Pulsar cluster deployed by docker:
+
+Service URL is `pulsar://pulsar:6650`, Admin URL is `http://pulsar:8080`.
+:::
+
+### Register ClickHouse DataNodes
+
+Click [DataNodes] -> [Create] on the page to register ClickHouse DataNodes.
+
+![Create Clickhouse Datanode](img/mysql_clickhouse/clickhouse_datanode.png)
```
Comment (EMsnap) [inline docs/quick_start/data_sync/mysql_clickhouse_example.md:61]: please provide a english version for this pic

File: docs/quick_start/data_sync/mysql_clickhouse_example.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+
+![Create Source](img/mysql_clickhouse/source.png)
+
+:::note
+- Please create the test.source_table database table in advance, the schema is: CREATE TABLE test.source_table (id INT PRIMARY KEY, name VARCHAR(50));
+:::
+
+### Create Data Sink
+
+In the data target, click [New] → [ClickHouse] to configure the name, library table, and created ck data node.
```
Comment (EMsnap) [inline docs/quick_start/data_sync/mysql_clickhouse_example.md:81]: library table -> table name ?

File: docs/quick_start/data_sync/mysql_clickhouse_example.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+done
+```
+
+Insert 1000 pieces of data in total:
+
+![Source_data](img/mysql_clickhouse/source_data.png)
+
+### Verify Data
+
+Then enter the ClickHouse container and view the library table data:
```
Comment (EMsnap) [inline docs/quick_start/data_sync/mysql_clickhouse_example.md:133]: ditto

File: docs/quick_start/data_sync/mysql_clickhouse_example.md (hunk 1 above)
Comment (haifxu) [reply docs/quick_start/data_sync/mysql_clickhouse_example.md:61]: Done.

File: docs/quick_start/data_sync/mysql_clickhouse_example.md (hunk 2 above)
Comment (haifxu) [reply docs/quick_start/data_sync/mysql_clickhouse_example.md]: Done.

File: docs/quick_start/data_sync/mysql_clickhouse_example.md (hunk 3 above)
Comment (haifxu) [reply docs/quick_start/data_sync/mysql_clickhouse_example.md]: Done.

**Review by vernedeng — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**